src/nvim/edit.c

```diff
@@ -7989,7 +7989,7 @@ static bool ins_esc(long *count, int cmdchar, bool nomove)
   may_trigger_modechanged();
   // need to position cursor again when on a TAB
   if (gchar_cursor() == TAB) {
-    curwin->w_valid &= ~(VALID_WROW|VALID_WCOL|VALID_VIRTCOL);
+    curwin->w_valid &= ~(VALID_WROW | VALID_WCOL | VALID_VIRTCOL);
```

Comment:
could we change `|` to don't have space? think the old looked better.

Reply:
No, not that I'm aware of.